core/src/main/scala/io/qbeast/core/transform/LinearTransformation.scala

```diff
@@ -89,41 +89,26 @@ case class LinearTransformation(
     }
   }
 
-  /**
-   * Merges two transformations. The domain of the resulting transformation is the union of this
-   * and the other transformation.
-   */
-  override def merge(other: Transformation): Transformation = {
-    other match {
-      case LinearTransformation(otherMin, otherMax, _, otherOrdering)
-          if orderedDataType == otherOrdering =>
-        LinearTransformation(min(minNumber, otherMin), max(maxNumber, otherMax), orderedDataType)
-      case IdentityTransformation(newVal, otherType) if orderedDataType == otherType =>
-        LinearTransformation(min(minNumber, newVal), max(maxNumber, newVal), orderedDataType)
-      case IdentityToZeroTransformation(newVal) =>
-        LinearTransformation(min(minNumber, newVal), max(maxNumber, newVal), orderedDataType)
-      case _ => this
-    }
+  override def isSupersededBy(other: Transformation): Boolean = other match {
+    case LinearTransformation(newMin, newMax, _, otherOrdering)
+        if orderedDataType == otherOrdering =>
+      gt(minNumber, newMin) || lt(maxNumber, newMax)
+    case IdentityTransformation(newVal, otherType) if otherType == orderedDataType =>
+      newVal != null && (gt(minNumber, newVal) || lt(maxNumber, newVal))
+    case _: EmptyTransformation => false
+    case _ => true
```

Comment:
Linear Transformation should be superseded by Hash?

Reply:
And is it superseded by a CDF of a different data type? (even though this casuistic is less likely to happen because of how we initialize the Transformations...)

Let's make this method (alongside with the `merge`) more explicit by including all the transformations available